src/editions.md

```diff
@@ -0,0 +1,39 @@
+# Editions
+
+Before we get into the details of the language itself, we need to cover the
```

Comment:
The general flow of the reference, as it stands today, is that you can jump in on any page. Unlike *The Book*, the reference assumes you know Rust well enough that you should be able to jump to any page and then link around to concepts you don't know. This paragraph seems to imply that the reference should be read from front to end, which is a change to that concept. If we don't want to change that, it'd be best to just remove this paragraph.